### ISSUE TITLE
elpy-format-code not finding yapf/autopep8 in rpc-venv

### ISSUE THREAD

**Issue by fgsedgwick** [jorgenschaefer/elpy#1730]
# Summary

elpy-config shows Autopep8 (1.4.4) and Yapf (0.28.0) installed in rpc-venv, but running elpy-format-code returns an error "Install yapf/autopep8 to format code."

# Steps to reproduce

Start with clean Elpy install, no RPC virtual environment. Allow Elpy to create RPC environment and populate it with required packaged. Open a .py file and turn elpy-mode on. Run elpy-format-code.

# My configuration
## OS
Mac OS Catalina (10.15.1)

## Result of `(elpy-config)`
```
Emacs.............: 26.1
Elpy..............: 1.32.0
Virtualenv........: None
Interactive Python: python 3.8.0 (/opt/local/bin/python)
RPC virtualenv....: rpc-venv (/Users/Forrest/.emacs.d/elpy/rpc-venv)
 Python...........: python3 3.7.3 (/Users/Forrest/.emacs.d/elpy/rpc-venv/bin/python3)
 Jedi.............: 0.15.1
 Rope.............: 0.14.0
 Autopep8.........: 1.4.4
 Yapf.............: 0.28.0
 Black............: 19.10b0
Syntax checker....: flake8 (/opt/local/bin/flake8)
```

**Comment by fgsedgwick** [comment]
Should elpy-config--package-available-p be using with-elpy-rpc-virtualenv-activated? It just calls elpy-rpc-python-command.

**Comment by fgsedgwick** [comment]
Calling elpy-(autopep8|yapf|black)-fix-code all work, it's just elpy-config--package-available-p that is failing.  It calls elpy-rpc-python-command; which I'm confused about. I thought that this command was only used to set up the virtual environment for the RPC, but that the RPC itself should be the python in rpc-env.  So if I understand this correctly, elpy-config--package-available-p should be using the rpc-venv python, not elpy-rpcs-python-command.

**Comment by galaunay** [comment]
You are completely right.
This occurrence of `elpy-rpc-python-command` slipped under my radar when implementing the RPC dedicated virtualenv.

I fixed it in c98e52b.
Please re-open if there is still something wrong.
